helper/consulutil/consul_ratelimit_publisher.go

```diff
@@ -46,6 +46,11 @@ type consulStore struct {
 	ctx context.Context
 }
 
+// GetKV returns the KV associated to the consul publisher
+func GetKV() *api.KV {
```

Comment:
hummm why?

Reply:
I need it in SetInstanceAttributeComplex function by instance